go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.3.1
 	github.com/hashicorp/go-azure-helpers v0.61.0
-	github.com/hashicorp/go-azure-sdk v0.20230918.1115907
+	github.com/hashicorp/go-azure-sdk v0.20230922.1111207
```

Comment:
Can we split this out into a separate PR please? We regenerate this regularly and need to ensure we don't create conflicts in the versions used.

Reply:
Ok, I will close this PR and reopen it after tests are running property.